api/account.go

```diff
@@ -37,8 +38,16 @@ func (a *API) getAccounts(c *gin.Context) {
 		return
 	}
 
+	request := requests.GetAccountsAPIRequest{
+		TokenIDs: tokenIDs,
+		EthAddr:  addr,
+		Bjj:      bjj,
+		FromItem: fromItem,
+		Limit:    limit,
+		Order:    order,
+	}
 	// Fetch Accounts from historyDB
-	apiAccounts, pendingItems, err := a.h.GetAccountsAPI(tokenIDs, addr, bjj, fromItem, limit, order)
+	apiAccounts, pendingItems, err := a.h.GetAccountsAPI(request)
```

Comment:
This is a subjective comment.  Have you considered calling the method without declaring the request variable?

Like this:
```go
	apiAccounts, pendingItems, err := a.h.GetAccountsAPI(requests.GetAccountsAPIRequest{
		TokenIDs: tokenIDs,
		EthAddr:  addr,
		Bjj:      bjj,
		FromItem: fromItem,
		Limit:    limit,
		Order:    order,
	})
```
It looks cleaner to me!